src/App.js

```diff
@@ -1,11 +1,23 @@
-import logo from './logo.svg';
+/* eslint-disable react/jsx-pascal-case */
 import './App.css';
-
+import { useState ,createContext} from 'react';
+import Main_head from './components/Main_head';
+import Header from './components/Header'
+import Proudects from './components/Proudects';
+import Footer from './components/Footer';
+export const ProductContxt = createContext(0)
 function App() {
+  const [clicked,setClicked] = useState(0)
```

Comment:
Since this is an integer, re-naming it to [noOfClicks,setNoOfClicks] makes more sense.

The word "clicked" indicates that it was either clicked or not, which works more with boolean types.

Reply:
done